ingestion/src/metadata/ingestion/source/pipeline/airflow/lineage_parser.py

```diff
@@ -112,14 +116,14 @@ def get_xlets_from_operator(
     :param xlet_mode: get inlet or outlet
     :return: list of tables FQN
     """
-    xlet = getattr(operator, xlet_mode)
+    xlet = getattr(operator, xlet_mode) if hasattr(operator, xlet_mode) else []
     xlet_data = parse_xlets(xlet)
```

Comment:
How have we resolved this comment?

Reply:
For reference: What happens if xlet is None? I think we'll get a ValueError on https://github.com/open-metadata/OpenMetadata/blob/main/ingestion/src/metadata/ingestion/source/pipeline/airflow/metadata.py#L363